library/src/main/scala/scala/scalajs/js/typedarray/TypedArrayBufferOps.scala

```diff
@@ -20,7 +20,7 @@ import java.nio._
  *  for some TypedArrayType, which depends on the type of elements in the
  *  buffer.
  */
-final class TypedArrayBufferOps[
+final class TypedArrayBufferOps[ // scalastyle:ignore
```

Comment:
It doesn't like that there is a whitespace after `[` (in this case, a new line).